scripts/entrypoint.sh

```
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+set -euo pipefail
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)